src/main/java/org/veupathdb/eda/binaryfiles/dumper/multi/Main.java

```diff
@@ -0,0 +1,49 @@
+package org.veupathdb.eda.binaryfiles.dumper.multi;
+
+import org.gusdb.fgputil.db.platform.SupportedPlatform;
+import org.gusdb.fgputil.db.pool.DatabaseInstance;
+import org.gusdb.fgputil.db.pool.SimpleDbConfig;
+import org.veupathdb.eda.binaryfiles.dumper.StudyDumper;
+import org.veupathdb.service.eda.ss.model.Study;
+import org.veupathdb.service.eda.ss.model.StudyOverview;
+import org.veupathdb.service.eda.ss.model.db.StudyFactory;
+import org.veupathdb.service.eda.ss.model.db.StudyProvider;
+import org.veupathdb.service.eda.ss.model.db.StudyResolver;
+
+import javax.sql.DataSource;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.gusdb.fgputil.runtime.Environment.getRequiredVar;
+
+public class Main {
```

Comment:
minor points (and these apply to my original main() method, in retrospect)
- use "APPDB_xxxx" for the env vars, not "APP_DB_xxxx"... those are "standard" in bob's world
- seems we should include APPDB_SCHEMA as a req'd var 
- if no args passed, print usage, including the req'd env vars